lightning/src/ln/channelmanager.rs

```diff
@@ -1203,6 +1204,36 @@ impl ChannelDetails {
 	}
 }
 
+/// We store payments that are pending resolution as `PendingOutboundPayment` in `ChannelManager`,
+/// and they can take on either one of the following states as defined in this enum.
+#[derive(Debug, PartialEq)]
+pub enum PendingPaymentStatus {
+	/// When a payment is still being sent and awaiting successful delivery.
+	Retryable,
```

Comment:
I think it's potentially confusing to call this case `Pending` since looking up the enum `PendingPaymentStatus` from the user's perspective already hints implicitly that these cases are meant to describe _how_ the payments are pending. 

I don't have an issue with calling this `Retryable` mostly because it suitably describes the complement of `Fulfilled` and `Abandoned, which express a "success but pending confirmation" and "user has explicitly given up, or payer will not longer _retry_" states respectively. thoughts?

Reply:
Brainstorming:
- what do we want to mean by "pending" in the enum type?
- is there another word that better describes what we'd like to convey (tracked? outstanding? revokable? unresolved?) such that pending could make sense for this variant?

Reply:
Yeah, agreed. The alternatives that I mentioned were more for the top-level type, letting us use "pending" for the variant. Not sold on "in-flight" given we'd overload the meaning. Indifferent to "recent" as I suppose it conveys what is represented but not specifically on what is defined to be "recent" without explanatory docs.

Reply:
resolved in fb0b1afc